instrumentation/opentelemetry-instrumentation-botocore/tests/test_botocore_bedrock.py

```diff
@@ -2519,3 +2529,165 @@ def get_anthropic_tool_config():
             },
         }
     ]
+
+
+@pytest.mark.skipif(
+    BOTO3_VERSION < (1, 34, 1), reason="InvokeAgent API not available"
+)
+@pytest.mark.vcr()
+def test_invoke_agent_with_completion(
+    span_exporter,
+    metric_reader,
+    bedrock_runtime_client,
```

Comment:
Do you still need the bedrock runtime client?

Reply:
no, though the create_agent scaffolding will need it still (for s3 and such) but it will be done elsewhere